src/main/java/com/carrotcreative/socialui/util/SocialMovementMethod.java

```diff
@@ -65,6 +66,10 @@ else if(url.startsWith(SOCIAL_UI_MENTION_SCHEME))
             mention = mention.replaceFirst(".*@", "");
             mHandler.handleMention(mention);
         }
+        else if(Patterns.EMAIL_ADDRESS.matcher(url).matches())
```

Comment:
Lolol I thought this was going to be more work, but I already had this proxy method :cat: .

Good work :laughing:  :+1: